Front/src/app/components/admin/add-edit-customer/add-edit-customer.component.html

```diff
@@ -0,0 +1,92 @@
+<app-loader *ngIf="loader"></app-loader>
+<div class="container-sm">
+    <main class="pb-3">
+        <div class="row h-50 ">
+            <h3 class="mb-4 mt-4 text-center">{{operacion}} Cliente</h3>
+            <div class="col-md-12 mb-3">
+                <form 
+                [formGroup]="formCustomer"
+                (ngSubmit)="AddEditCustomer()"
+                class="row g-3">
+
+                    <div class="col-md-4 mb-3 ">
+                        <label for="Nit" class="control-label">Nit</label>
+                        <input type="text" class="form-control mt-2" placeholder="Nit" id="Nit" formControlName="nit">
+                        <div class="">
+                            <span *ngIf="
+                                    formCustomer.get('nit')?.hasError('required') && 
+                                    formCustomer.get('nit')?.touched"
+                            class="form-text m-2 text-danger"> El nit es<strong> requerido </strong> </span>
+                        </div>
+                    </div>
+                    
+                    <div class="col-md-4 mb-3">
+                        <label for="Nombre" class="control-label">Nombre</label>
+                        <input type="text" class="form-control mt-2" placeholder="Nombre" id="Nombre" formControlName="name">
+                        <div>
+                            <span *ngIf="
+                            formCustomer.get('name')?.hasError('required') && 
+                            formCustomer.get('name')?.touched"
+                             class="form-text m-2 text-danger"> El nombre es<strong> requerido </strong> </span>
+                        </div>
+                    </div>
+
+                    <div class="col-md-4 mb-3">
+                        <label for="Nombre" class="control-label">Apellido</label>
+                        <input type="text" class="form-control mt-2" placeholder="Apellido" id="Apellido" formControlName="lastname">
+                        <div>
+                            <span *ngIf="
+                            formCustomer.get('lastName')?.hasError('required') && 
+                            formCustomer.get('lastNsame')?.touched"
+                            class="form-text m-2 text-danger"> El teléfono es<strong> requerido </strong> </span>
+                        </div>
+                    </div>
+
+                    <div class="col-md-4 mb-3">
+                        <label for="Nombre" class="control-label">Direccion</label>
```

Comment:
Same issue with the for attribute for all the rest. It should be pointing to the input id and the input id should be in English, only visible text for the user will be in Spanish.
```suggestion
                        <label for="Address" class="control-label">Direccion</label>
```